internal/pkg/utils/testproject/snapshot.go

```diff
@@ -201,8 +193,14 @@ func (p *Project) NewSnapshot() (*fixtures.ProjectSnapshot, error) {
 
 	snapshot.Buckets = make([]*fixtures.Bucket, 0)
 	for _, b := range bucketsMap {
+		sort.Slice(b.Tables, func(i, j int) bool {
+			return b.Tables[i].ID.String() < b.Tables[j].ID.String()
+		})
 		snapshot.Buckets = append(snapshot.Buckets, b)
 	}
+	sort.Slice(snapshot.Buckets, func(i, j int) bool {
+		return snapshot.Buckets[i].ID.String() < snapshot.Buckets[j].ID.String()
+	})
```

Comment:
Takže buckety i tabulky v nich seřadím podle ID, jestli to chápu dobře.

Reply:
Ano.